test/user.test.js

```diff
@@ -2,16 +2,17 @@ const
     request = require('supertest'),
     app = require('../server');
 
-describe("Create forum user dummy test", function() {
-    it("should return: { dummyTest: 'userCreate() dummy test passes' }", function(done) {
+describe("Create forum user test", function() {
+    it("should return: status 200", function(done) {
```

Comment:
```suggestion
    it("should return: status 201", function(done) {
```
This is the status code that should be returned, and is being returned in my suggestions.